doc/connectivity/resting_state_networks.rst

```diff
@@ -91,3 +95,74 @@ Finally, we can plot the map for different ICA components separately:
    displayed on your computer might not match those of the documentation. For
    a fair representation, you should display all components and
    investigate which one resemble those displayed above.
+
+Beyond ICA : Dictionary learning
+================================
+
+Recent work has shown that dictionary learning based techniques outperform
+ICA in term of stability and constitutes a better first step in a statistical
+analysis pipeline.
+Dictionary learning in neuro-imaging seeks to extract a few representative
+temporal elements along with their sparse spatial loadings, which constitute
+good extracted maps.
+
+.. topic:: **References**
+
+   * Gael Varoquaux et al. `Multi-subject dictionary learning to segment an atlas of brain spontaneous activity <http://hal.inria.fr/inria-00588898/en/>`_,
+     IPMI 2011, pp. 562-573, Lecture
+     Notes in Computer Science
+
+Applying DictLearning
+---------------------
+
+:class:'DictLearning' is a ready-to-use class with the same interface as CanICA.
+Sparsity of output map is controlled by a parameter alpha: using a
+larger alpha yields sparser maps.
+
+.. literalinclude:: ../../examples/connectivity/plot_compare_resting_state_decomposition.py
+    :start-after: # Dictionary learning
+    :end-before:# CanICA
+
+We can fit both estimators to compare them
+
+.. literalinclude:: ../../examples/connectivity/plot_compare_resting_state_decomposition.py
+    :start-after: # Fitting both estimators
+    :end-before: # Visualize the results
+
+Visualizing the results
+-----------------------
+
+4D plotting offers an efficient way to compare both resulting outputs
+
+.. literalinclude:: ../../examples/connectivity/plot_compare_resting_state_decomposition.py
+    :start-after: # Visualize the results
+
+.. |left_img| image:: ../_images/sphx_glr_plot_compare_resting_state_decomposition_001.png
```

Comment:
In order to be compatible with the changes that have been going on in sphinx-gallery, you should probably a pattern something like this here:

../auto_examples/connectivity/images/sphx_glr_plot_canica_resting_state_003.png

Same thing for the png below.